pom.xml

```diff
@@ -98,7 +98,7 @@
     <commons-logging.version>1.2</commons-logging.version>
     <commons-csv.version>1.0</commons-csv.version>
     <sqlline.version>1.1.8</sqlline.version>
-    <guava.version>12.0.1</guava.version>
+    <guava.version>13.0.1</guava.version>
```

Comment:
Yes, it's likely an issue, but not positive. @apurtell might know. Would you mind opening a Tephra JIRA to explore our options? Where is the dependency on 13.0.1? Is it on the server-side in the transaction manager?